twilio/Package.md

```diff
@@ -1,337 +1,18 @@
-    };
-      
-   //Twilio Client configuration
-    twilio:TwilioConfiguration twilioConfig = {
-        accountSId: accountSId,
-        authToken: authToken
-    };
-
-    //Twilio Client
-    twilio:Client twilioClient = check new (twilioConfig);
-
-    //Make voice Call remote function is called by the twilio client
-     var details = twilioClient->makeVoiceCall(fromMobile, toMobile, voiceInput);
-
-    //Response is printed as log messages
-    if (details is twilio:VoiceCallResponse) {
-        log:printInfo("Message Detail: " + details.toString());
-    } else {
-        log:printInfo(details.message());
-    }
-}
-
-```   
-
-### Get a message
-This section shows you how to get a message details  from your account. you need to provide message sid to retreive the details from the message list of your account. If the request is successful, it will send the MessageResourceResponse record else an error message with the details.
-Sample is available at: samples/client samples/getMessage.bal
-```ballerina
-import ballerina/log;
-import ballerinax/twilio;
-configurable string accountSId = ?;
-configurable string authToken = ?;
-public function main() {
-    //Twilio client configuration
-    twilio:TwilioConfiguration twilioConfig = {
-        accountSId: accountSId,
-        authToken: authToken
-    };
-
-    //Twilio client
-    twilio:Client twilioClient = new(twilioConfig);
-    
-    //Set Message resource SID to get themessage detial
-    string messageSid = "<Add Mesaage SID>";
-
-    //Get SMS remote function is called by thetwilio client
-    var details = twilioClient->getMessag(messageSid);
-    
-    //Response is printed as log messages
-    if (details istwilio:MessageResourceResponse) {
-        log:printInfo("Message Detail: " +details.toString());
-    } else {
-        log:printInfo(details.message());
-    }
-}
-```
-# Building from the source
-1. You need to obtain a twilio Account SID, an Auth Token and a valid twilio phone number to use the twilio connector operations.
-2. Clone the repository and Be sure to change to the branch which has the name as the Ballerina version. Eg: slAlpha5
-    * Note: You need to install the relevant Ballerina version before going to the next step.
-3. First, If you want to run tests, you will need to add to a Config.toml files to the client module test directory with the following credentials.
-```ballerina
-[ballerinax.twilio]
-twilioAccountSid ="<ACCOUNT SID>"
-twilioAuthToken ="<AUTH TOKEN>"
-fromNumber ="<YOUR TWILIO NUMBER>"
-toNumber ="RECIPIENT NUMBER"
-test_message ="<SMS MESSAGE>"
-twimlUrl ="<TWIML LINK>"
-fromWhatsappNumber="<TWILIO WHATSAPP NUMBER>"
-```
-4. Run `./gradlew build` to build the java wrapper.
-
-5. Run `bal build twilio` to build with tests or else Run `bal build --skip-tests twilio`.
+Connects to Twilio from Ballerina
+
+### Package Overview
+The `ballerinax/twilio` is a [Ballerina](https://ballerina.io/) connector for twilio.
```

Comment:
Suggestion: "The `ballerinax/twilio` is a [Ballerina](https://ballerina.io/) connector for Twilio."